presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleNodeSelector.java

```diff
@@ -215,6 +226,11 @@ public SplitPlacementResult computeAssignments(Set<Split> splits, List<RemoteTas
         InternalNode chosenNode = null;
         for (int i = 0; i < candidateNodes.size(); i++) {
             InternalNode node = candidateNodes.get(i);
+            if (node.getNodeStatus() == DEAD) {
+                // Node is down. Do not schedule split
```

Comment:
shall we add have metric counter to track it?

Reply:
I think we already have metrics to 
- track the deadNodes  (you used it recently in your dashboard)
- to track node selection. primary vs non-primary in `NodeSelectionStats` 

So i dont think we need any new metric. WDYT ?

Reply:
I don't have strong preference. 
1. The current node alive dashboard mention how many nodes are down.
2. primary vs non-primary vs random does not tell the reason why it chose. Busy node, dead node?
3. A new metric would clearly demonstrate the impact due to dead node selection.